src/Simulation/Native/src/simulator/qir.cpp

```diff
@@ -0,0 +1,504 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/*=============================================================================
+    QIR assumes a single global execution context.
+    To support the dispatch over the qir-bridge, the clients must register their
+    Microsoft::Quantum::IRuntimeDriver* first.
```

Comment:
Is this still accurate?

Reply:
Good catch! No, that is no longer accurate. I'll update the comment.